modules/ossm-deploying-multiple-control-planes.adoc

```diff
@@ -0,0 +1,191 @@
+// Module included in the following assemblies:
+// install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploying-multiple-control-planes_{context}"]
+= Deploying multiple control planes
+:context: ossm-deploying-multiple-control-planes
+
+This procedure explains how to configure a cluster to host two control planes, each managing independent service mesh data planes. The configuration involves creating separate `{istio}` resources with unique names in independent {istio} system namespaces. Each `{istio}` resource will use a unique revision name, which identifies the control planes, workloads, or namespaces associated with it. These revision names, applied using injection or `istio.io/rev` labels determine which control plane will inject the sidecar proxy into the application pods.
+
+Each `{istio}` resource must also configure discovery selectors to specify which namespaces the {istio} control plane observes. Only namespaces with labels that match the configured discovery selectors can join the mesh. Additionally, discovery selectors determine which control plane creates the `istio-ca-root-cert` config map in each namespace, which is used to encrypt traffic between services with mutual TLS within each mesh.
+
+When adding an additional {istio} control plane to a cluster with an existing control plane, ensure that the existing `{istio}` instance has discovery selectors configured to avoid overlapping with the new control plane. Run the following command to check for existing `{istio}` instances:
+
+[source,terminal]
+----
+$ oc get istios
+----
```

Comment:
Commands aren't typically present in module descriptions. Could you find a way to move this into the prereq or procedure blocks?